qemu/tests/blk_stream.py

```diff
@@ -15,7 +15,7 @@ class BlockStream(block_copy.BlockCopy):
 
     def __init__(self, test, params, env, tag):
         super(BlockStream, self).__init__(test, params, env, tag)
-        self.base_image = self.image_file
+        self.base_image = None
```

Comment:
Set base to guest image_file only when call `set_backingfile()` function. and  `base_image = None` is the default value in `qemu_monitor.block_stream`,  So we can delete it here,   add in it to ` class BlockStreamCheckBackingfile` is enough.

Reply:
No we can not, below start()function in this class pass this parameter to block_stream command. So have to keep it in __init__ as None.

Reply:
oh~, yes, you are right, start() is used not only for backing file related test. we must keep it. thanks !